src/styles/blocks/card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  background: $white_color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 32px auto 40px;
+  }
+
+  &__name {
+    font-weight: 500;
+    font-size: 12px;
+    color: $dark-color-header;
+    display: flex;
+    width: 166px;
+    height: 36px;
+    line-height: 18px;
+    margin: 0 auto 4px;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $lite-grey-clor;
+    width: 98px;
```

Comment:
do not hardcode `width & height` of the elements, if u'll remove this properties nothing will change in this case)